src/clr-angular/forms/select/select.spec.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+import { ClrSelect } from './select';
+import { ClrSelectContainer } from './select-container';
+
+import { TemplateDrivenSpec, ControlInvalidSpec, ReactiveSpec } from '../tests/control.spec';
+
+@Component({
+  template: `
+    <select clrSelect></select>
+    `,
+})
+class InvalidUseTest {}
+
+@Component({
+  template: `
+    <select clrSelect name="model" class="test-class" [(ngModel)]="model"></select>
+    `,
+})
+class TemplateDrivenTest {}
+
+@Component({
+  template: `
+  <div [formGroup]="example">
+    <select clrSelect name="model" class="test-class" formControlName="model"></select>
```

Comment:
QQ: does it matter it has both `name` and `formControlName`? I am assuming `formControlName` would override the static name property attribute. 

Also, it's `name="model"` in both `TemplateDrivenTest` and `ReactiveTest` templates, which is not binding. Was it for purpose?